From 07931ec2f127ce2408690cc85399ca750ca12657 Mon Sep 17 00:00:00 2001
From: FooIbar <118464521+FooIbar@users.noreply.github.com>
Date: Thu, 19 Jun 2025 00:03:46 +0800
Subject: [PATCH] Disable runtime detection of NEON support

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d7e8963f..9dda3bb1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -181,7 +181,7 @@ endif()
 
 # ##############################################################################
 # Android only.
-if(ANDROID)
+if(0)
   include_directories(${ANDROID_NDK}/sources/android/cpufeatures)
   add_library(cpufeatures-webp STATIC
               ${ANDROID_NDK}/sources/android/cpufeatures/cpu-features.c)
-- 
2.45.2

