diff --git a/fat-arm64.c b/fat-arm64.c
index aec99f66..62b3adea 100644
--- a/fat-arm64.c
+++ b/fat-arm64.c
@@ -43,9 +43,14 @@
 #if defined(__linux__) && defined(__GLIBC__) && defined(__GLIBC_PREREQ)
 # if __GLIBC_PREREQ(2, 16)
 #  define USE_GETAUXVAL 1
-#  include <asm/hwcap.h>
-#  include <sys/auxv.h>
 # endif
+#elif __ANDROID_API__ >= 18
+# define USE_GETAUXVAL 1
+#endif
+
+#if USE_GETAUXVAL
+# include <asm/hwcap.h>
+# include <sys/auxv.h>
 #elif defined(__OpenBSD__)
 # include <sys/sysctl.h>
 # include <machine/cpu.h>
